browser/ui/views/frame/brave_browser_view.h

```diff
@@ -97,6 +104,9 @@ class BraveBrowserView : public BrowserView {
 
   static void SetDownloadConfirmReturnForTesting(bool allow);
 
+  absl::optional<int> SetAccelerator(int command_id,
```

Comment:
optional: RegisterAcceleratorToFocusManager() would be a descriptive name.
And it'd be good to have comment about the returned value here ;)

Reply:
Good call!